src/servers/ZoneServer2016/entities/npc.ts

```diff
@@ -232,73 +202,136 @@ export class Npc extends BaseFullCharacter {
             weight: 10
           }
         ];
+        this.isZombie = true;
         break;
       case 9253:
       case 9002:
         this.nameId = StringIds.DEER;
-        this.requiredHarvestItems = [MeleeTypes.KNIFE];
         this.rewardItems = [
-          {
-            itemDefId: 0,
-            weight: 40
-          },
           {
             itemDefId: Items.MEAT_VENISON,
             weight: 30
           },
           {
             itemDefId: Items.ANIMAL_FAT,
-            weight: 15
-          },
-          {
-            itemDefId: Items.DEER_SCENT,
-            weight: 5
+            weight: 20
           },
           {
             itemDefId: Items.DEER_BLADDER,
             weight: 10
           }
         ];
+        this.isWildlife = true;
         break;
       case 9003:
         this.nameId = StringIds.WOLF;
-        this.requiredHarvestItems = [MeleeTypes.KNIFE];
         this.rewardItems = [
-          {
-            itemDefId: 0,
-            weight: 45
-          },
           {
             itemDefId: Items.MEAT_WOLF,
-            weight: 40
+            weight: 30
           },
           {
             itemDefId: Items.ANIMAL_FAT,
-            weight: 15
+            weight: 20
           }
         ];
+        this.isWildlife = true;
         break;
       case 9187:
         this.nameId = StringIds.BEAR;
-        this.requiredHarvestItems = [MeleeTypes.KNIFE];
         this.rewardItems = [
-          {
-            itemDefId: 0,
-            weight: 45
-          },
           {
             itemDefId: Items.MEAT_BEAR,
             weight: 40
           },
           {
             itemDefId: Items.ANIMAL_FAT,
-            weight: 15
+            weight: 20
           }
         ];
+        this.isWildlife = true;
         break;
     }
   }
 
+  OnPlayerSelect(server: ZoneServer2016, client: ZoneClient2016) {
+    if (!this.isAlive && client.character.hasItem(Items.SKINNING_KNIFE)) {
+      server.utilizeHudTimer(client, this.nameId, 5000, 0, () => {
+        switch (this.actorModelId) {
+          case 9510:
```

Comment:
We don't have an enum for them ? If not we should do one

Reply:
Done 👍